source/_integrations/dominos.markdown

```diff
@@ -1,22 +1,22 @@
 ---
-title: Dominos Pizza
-description: Instructions on how to setup Dominos Pizza ordering within Home Assistant.
+title: Domino's Pizza
```

Comment:
The title is synced from the code base, changing it here will be reverted automatically.